### PR TITLE
feat: Display dummy application using ember-freestyle

### PR DESCRIPTION
**Pull request by KaufmanDeveloper**
This will allow us to use ember-freestyle's usage API to develop and display our UI components.

**To test:** Serve the application and confirm the display of our dummy app is backwards compatible with our current version and that the proper controls have been implemented for component properties.

### CODE REVIEW

File: tests/dummy/app/components/freestyle/code-block.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/* eslint-disable ember/no-empty-glimmer-component-classes */
```
Comment (TSenter) [inline tests/dummy/app/components/freestyle/code-block.js:1]: You can delete this file.

File: tests/dummy/app/components/freestyle/error-pages.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/* eslint-disable ember/no-empty-glimmer-component-classes */
```
Comment (TSenter) [inline tests/dummy/app/components/freestyle/error-pages.js:1]: This file can be deleted.

**Review by TSenter — CHANGES_REQUESTED** [review]
Does everything have to be created as its own Freestyle component? Can you not copy the templates for those components into the route template, and the JavaScript code into the controllers?

**Comment by KaufmanDeveloper** [comment]
> Does everything have to be created as its own Freestyle component? Can you not copy the templates for those components into the route template, and the JavaScript code into the controllers?

You could, I would personally opt for components however so we can reuse them if for example we want a form component to be on its own route. I also tend to avoid using controllers where a component can be used instead.

File: tests/dummy/app/components/freestyle/code-block.js (hunk 1 above)
Comment (KaufmanDeveloper) [reply tests/dummy/app/components/freestyle/code-block.js]: Thanks for the catch! https://github.com/knoxville-utilities-board/ember-nrg-ui/pull/494/commits/20e7053e7b9508edfbaf36bdb0584690d99841e7

File: tests/dummy/app/components/freestyle/error-pages.js (hunk 2 above)
Comment (KaufmanDeveloper) [reply tests/dummy/app/components/freestyle/error-pages.js]: https://github.com/knoxville-utilities-board/ember-nrg-ui/pull/494/commits/20e7053e7b9508edfbaf36bdb0584690d99841e7

**Review by KeithClinard — APPROVED** [review]

**Review by TSenter — CHANGES_REQUESTED** [review]

**Review by TSenter — APPROVED** [review]

**Pull request merged**